### PR TITLE
Gzip assets for production build.

### PR DESCRIPTION
**Pull request by rcorral**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

File: package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
     "babel-preset-react-hmre": "1.1.1",
     "babel-register": "6.8.0",
     "babel-runtime": "6.6.1",
+    "compression-webpack-plugin": "^0.3.1",
```
Comment (MatApple) [inline package.json]: `^`

**Comment by MatApple** [comment]
Just the one comment then 🚢. Epic compression!! css reduced by ~90%, js by ~75%

File: package.json (hunk 1 above)
Comment (rcorral) [inline package.json]: Done.

**Pull request merged**